### PR TITLE
Hw20

### PR DESCRIPTION
**Pull request by andrjuxa201185**
@iliatcymbal PTAL

### CODE REVIEW

File: src/components/tabs/Tabs.js [hunk 1]
```diff
@@ -1,9 +1,27 @@
+import PropTypes from 'prop-types';
 import { TabNav } from './TabNav';
 
 export const Tab = () => null;
 
 export class Tabs extends Component {
-  state = { id: 0 }
+  // eslint-disable-next-line react/destructuring-assignment
+  state = { id: this.props.selectedIndex } // <-- please tell me, how I can this destructuring???
```
Comment (iliatcymbal) [inline src/components/tabs/Tabs.js]: - you should ask eslint authors)
- actually, you have `getDerivedStateFromProps` which is set state on first rendering

File: src/components/tabs/Tabs.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+
+  static getDerivedStateFromProps(nextProps, state) {
+    if (nextProps.selectedIndex.length !== state.id.length) {
```
Comment (iliatcymbal) [inline src/components/tabs/Tabs.js]: Why do you compare length? It doesn't work

File: src/components/images/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export * from './images';
```
Comment (iliatcymbal) [inline src/components/images/index.js]: there is no `images.js` file

**Comment by andrjuxa201185** [comment]
@iliatcymbal, Is this right?

**Comment by iliatcymbal** [comment]
https://github.com/andrjuxa201185/andrey_podgorniy_shop/blob/740b5a5bcaf4b5d49f199a8f13c21543e587aec7/src/components/images/index.js#L1

there is still no `images.js` file

**Pull request merged**